src/main/java/org/cbioportal/cdd/service/internal/ClinicalAttributeMetadataCache.java

```diff
@@ -93,10 +101,14 @@ public void resetCache() {
                 logger.error("resetCache(): failed to pull overrides from repository");
             }
             consecutiveFailedAttempts += 1;
-            if (consecutiveFailedAttempts >= MAX_FAILED_ATTEMPTS) {
-                logger.error("resetCache(): failed to pull from repository " + consecutiveFailedAttempts +  " times, emptying caches");
+            if (consecutiveFailedAttempts >= MAX_FAILED_ATTEMPTS || force) {
                 clinicalAttributeCache = null;
                 overridesCache = null;
+                if (consecutiveFailedAttempts >= MAX_FAILED_ATTEMPTS) {
```

Comment:
here we are using the consecutiveFailedAttempts to distinguish between a forced pull and a scheduled pull failure. But since nothing resets consecutiveFailedAttempts except for a successful reload of cache, after three failed force refreshes, the log will start reporting the non-force version of the failure. Perhaps the condition tested should just be "if (force)" and then switch the order of log statements.